<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
"http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=gb2312">
<META HTTP-EQUIV="Pragma" CONTENT="no-cache">
<meta name="keywords" content="3D genome browser, Hi-C browser, ChIA-PET browser, Capture Hi-C browser.">
<meta name="description" content="3D genome browser, Hi-C browser, ChIA-PET browser, Capture Hi-C browser.">
<meta name="keywords" content="mouse ENCODE project">
<link rel="stylesheet" href="style2.css" type="text/css" media="all">
<title>3D Genome Browser</title>
</head>

<body>
<div id="wrap">
<div id="header">
  <a href="http://yuelab.org" target="_blank"><img src="/img/yue_lab_logo2.jpg" border="0"></a>
</div>
<ul id="nav">
        <li><a href="index.html">Intro</a></li>
        <li><a href="view.php">Hi-C</a></li>
        <li><a href="virtual4c.php">Virtual 4C</a></li>
        <li><a href="chiapet.php">ChIA-PET & Similar</a></li>
        <li><a href="chic.php">Capture Hi-C</a></li>
        <li><a href="compare.php">Compare Hi-C</a></li>
        <li><a href="iview.php">Inter-chrom</a></li>
        <li><a href="tutorial.html">Tutorial</a></li>
        <li><a href="publications.html">Resource</a></li>

</ul>
<div id="content">
<h1> How to Cite Our Work</h1>
<img src="/img/divider_horizontal_4px.gif">
<p> 
<strong>Figures generated in this website can be directly used in your publication</strong>. If you do, please cite our work, "The 3D Genome Browser: a web-based browser for visualizing 3D genome organization and long-range chromatin interactions." <a href="http://biorxiv.org/content/early/2017/02/27/112268" target=_bland>http://biorxiv.org/content/early/2017/02/27/112268</a>, Biorxiv, 2017.
<p>
<h1> Downloads  </h1>
<img src="/img/divider_horizontal_4px.gif">
<p>
<a href="./downloads/hg19.TADs.zip">TADs in hg19 </a>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <a href="./downloads/hg38.TADs.zip">TADs in hg38 </a>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <a href="./downloads/mm9.TADs.zip">TADs in mm9 </a>
<p>
Note: the TADs coordinates provided here are predicted using our in-house pipeline from Dixon et al. Nature 2012 and the list of tissue/cell type will be constantly updated as we add more data. Please contact us if you have any quesitons. 
<p>
<h1> Source of data available for visualization</h1>
<img src="/img/divider_horizontal_4px.gif">
<p>
We are hosting all the Hi-C and ChIA-PET datasets produced by the <strong> <a href="http://www.encodeproject.org/">ENCODE Consortium.</a> </strong>

You can also visualize and explore the chromatin interaction data from the following publications. If you have suggestions of datasets to be added to this browser, please contact us.
<p>
1.&nbsp; Lieberman-Aiden, E., Nynke L. van Berkum, Williams, L., Imakaev, M., Ragoczy, T., Telling, A., Amit, I., Lajoie, B.R., Sabo, P.J., Dorschner, M.O., Sandstrom, R., Bernstein, B., Bender, M.A., Groudine, M., Gnirke, A., Stamatoyannopoulos, J., Mirny, L.A., Lander, E.S. & Dekker, J. Comprehensive Mapping of Long-Range Interactions Reveals Folding Principles of the Human Genome. <strong>Science. 2009. </strong> 326(5950):289-293. <a href="http://www.ncbi.nlm.nih.gov/pubmed/19815776">Pubmed</a>  <a href="http://science.sciencemag.org/content/326/5950/289.full">Paper</a>  <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE18199">Dataset</a>
<p>
2. &nbsp;Dixon JR, Selvaraj S, Yue F, Kim A, Li Y, Shen Y, Hu M, Liu JS, Ren B. Topological domains in mammalian genomes identified by analysis of chromatin interactions. <strong>Nature. 2012</strong>;485(7398):376-380.  <a href="http://www.ncbi.nlm.nih.gov/pubmed/22495300">Pubmed</a>  <a href="http://www.nature.com/nature/journal/v485/n7398/full/nature11082.html">Paper</a>  <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35156">Dataset</a>
<p>
3. &nbsp;Shen, Y., Yue, F., McCleary, D., Ye, Z., Edsall, L., Kuan, S., Wagner, U., Dixon, J., Lee, L., Lobanenkov, V. & Ren, B. A map of the cis-regulatory sequences in the mouse genome". <strong>Nature. 2012</strong>;488(7409):116-120. <a href="http://www.nature.com/nature/journal/v488/n7409/full/nature11243.html"> Paper</a> <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29184">Dataset</a>
<p>
4. &nbsp;Rao, S.S.P., Huntley, M.H., Durand, N.C., Stamenova, E.K., Bochkov, I.D., Robinson, J.T., Sanborn, A.L., Machol, I., Omer, A.D., Lander, E.S. & Aiden-Lieberman, E. 2014,. A 3D Map of the Human Genome at Kilobase Resolution Reveals Principles of Chromatin Looping. <strong>Cell. 2014</strong>;159(7):1665-1680. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25497547">Pubmed</a>  <a href="http://www.sciencedirect.com/science/article/pii/S0092867414014974">Paper</a>  <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63525">Dataset</a>
<p>
5. &nbsp;Heidari, N., Phanstiel, D.H., He, C., Grubert, F., Jahanbani, F., Kasowski, M., Zhang, M., Snyder, M.P. Genome-wide map of regulatory interactions in the human genome. <strong> Genome Research. 2014</strong> Dec; 24(12): 1905-1917. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4248309/">Pubmed</a>
<p>
6. &nbsp;Dixon JR, Jung I, Selvaraj S, Shen Y, Antosiewicz-Bourget JE, Lee AY, Ye Z, Kim A, Rajagopal N, Xie W, Diao Y, Liang J, Zhao H, Lobanenkov VV, Ecker JR, Thomson JA, Ren B. Chromatin architecture reorganization during stem cell differentiation. <strong>Nature. 2015</strong>;518(7539):331-336. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25693564">Pubmed</a>  <a href="http://www.nature.com/nature/journal/v518/n7539/full/nature14222.html">Paper</a>  <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52457">Dataset</a> 
<p>
7. &nbsp;Leung, D., Jung, I., Rajagopal, N., Schmitt, A., Selvaraj, S., Lee, A. Y. & Ren, B. Integrative analysis of haplotype-resolved epigenomes across human tissues. <strong>Nature. 2015</strong>;518(7539):350-354. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25693566">Pubmed</a>  <a href="http://www.nature.com/nature/journal/v518/n7539/full/nature14217.html">Paper</a>  <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE58752">Dataset</a>
<p>
8. &nbsp;Nagano, T., Lubling, Y., Yaffe, E., Wingett, S. W., Dean, W., Tanay, A., & Fraser, P. Single-cell hi-C for genome-wide detection of chromatin interactions that occur simultaneously in a single cell. <strong>Nature Protocols. 2015</strong>;10(12):1986-2003.
<p>
9. &nbsp; Mifsud, B., Tavares-Cadete, F., Young, A., Sugar, R., Schoenfelder, S., Ferreira, L., Wingett, S., Andrews, S., Grey, W., Ewels, P., Herman, B., Happe, S., Higgs, A., LeProust, E., Follows, G., Fraser, P., Luscombe, N. & Osborne, C. Mapping long-range promoter contacts in human cells with high-resolution capture Hi-C. <strong>Nature Genetics. 2015</strong>;47(6):598-606. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25938943">Pubmed</a>  <a href="http://www.nature.com/ng/journal/v47/n6/full/ng.3286.html">Paper</a>  <a href="http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2323/files/">Dataset</a>
<p>
10. &nbsp; Tang, Z., Luo, O., Li, G., Li, X., Zheng, M., Zhu, J., Szalaj, P., Trzaskoma, P., Magalska, A., Wlodarczyk, J., Ruszczycki, B., Michalski, P., Piecuch, E., Wang, P., Wang, D., Tian, S., Penrad-Mobayed, M., Sachs, L., Ruan, Y., Ruan, X., Wei, C., Liu, E., Wilczynski, G. & Plewczynski, D. CTCF-Mediated Human 3D Genome Architecture Reveals Chromatin Topology for Transcription. <strong> Cell. 2015</strong>, 163(7), p1611-1627. <a href="http://www.cell.com/cell/references/S0092-8674(15)01504-4">Paper</a>
<p>
11. &nbsp;Schmitt, A., Hu, M., Jung, I., Xu, Z., Qiu, Y., Tan, C., Li, Y., Lin, S., Lin, Y., Barr, C. & Ren, B. A Compendium of Chromatin Contact Maps Reveals Spatially Active Regions in the Human Genome. <strong>Cell Reports. 2016</strong>;17(8):2042-2059. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27851967">Pubmed</a>  <a href="http://www.cell.com/cell-reports/abstract/S2211-1247(16)31481-4">Paper</a>  <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87112">Dataset</a>
<p>
12. &nbsp;Javierre, B., Burren, O., Wilder, S., Kreuzhuber, R., Hill, S., Sewitz, S., Cairns, J., Wingett, S., V&aacute;rnai, C., Thiecke, M., Burden, F., Farrow, S., Cutler, A., Rehnstr&ouml;m, K., Downes, K., Grassi, L., Kostadima, M., Freire-Pritchett, P., Wang, F., Stunnenberg, H.G., Todd, J.A., Zerbino, D.R., Stegle, O., Ouwehand, W.H., Frontini, M., Wallace, C., Spivakov, M. & Fraser, P., Lineage-Specific Genome Architecture Links Enhancers and Non-coding Disease Variants to Target Gene Promoters. <strong>Cell. 2016</strong>;167(5):1369-1384.e19. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27863249">Pubmed</a>  <a href="http://www.cell.com/cell/abstract/S0092-8674(16)31322-8">Paper</a>
<p>
13. &nbsp;Fang, R., Yu, M., Li, G., Chee, S., Liu, T., Schmitt, A.D., and Ren, B. Mapping of long-range chromatin interactions by proximity ligation-assisted ChIP-seq.  <strong>Cell Res. 2016 </strong> Dec;26(12):1345-1348. <a href="http://www.nature.com/cr/journal/v26/n12/full/cr2016137a.html">Paper</a> <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE86150"> Dataset </a>
<p>
14. &nbsp;Mumbach, M.R., Rubin, A.J., Flynn, R.A., Dai, C., Khavari, P.A., Greenleaf, W.J. & Chang, H.Y. 2016, HiChIP: efficient and sensitive analysis of protein-directed genome architecture. <strong>Nature Methods.2016</strong>;13(11):919-922. <a href="http://www.nature.com/nmeth/journal/v13/n11/full/nmeth.3999.html"> Paper </a> 
<p>
15. &nbsp;Won, H., Torre-Ubieta, L., Stein, J.L., Parkshak, N.N., Huang, J., Opland, C.K., Gandal, M.J., Sutton, G.J., Hormozdiari, F., Lu, D., Lee, C., Eskin, E., Voineagu, I., Ernst, J., & Geschwind, D.H., Chromosome conformation elucidates regulatory relationships in developing human brain. <strong> Nature. 2016</strong>,538, 523-527. <a href="http://www.nature.com/nature/journal/v538/n7626/abs/nature19847.html"> Paper</a> <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE77565"> Dataset </a>
<p>
16. &nbsp; Jin, F., Li, Y., Dixon, J., Selvaraj, S., Ye, Z., Lee, A., Yen, C., Schmitt, A., Espinoza, C., & Ren, B. A high-resolution map of the three-dimensional chromatin interactome in human cells. <strong> Nature 2013 </strong>, 503(7475), 290-4.
<p>
17.&nbsp; Rubin, A., Barajas, B., Furlan-Magaril, M., Lopez-Pajares, V., Mumbach, M., Howard, I., .., Chang, H., Fraser, P., Khavari, P. Lineage-specific dynamic and pre-established enhancer-promoter contacts cooperate in terminal differentiation. <strong>Nat. Genet 2017 </strong>, 49(10), 1522-1528.

<p>
18. Ramirez, F., Lingg, T., Toscano, S., Lam, K., Georgiev, P., Chung, H., Lajoie, B., de Wit, E., Zhan, Y., de Laat, W., Dekker, J., Manke, T., Akhtar, A. High-Affinity Sites Form an Interaction Network to Facilitate Spreading of the MSL Complex across the X Chromosome in Drosophila. <strong>Mol. Cell 2015 </strong>, 60(1), 146-62.
<p>
19. Phanstiel DH, Van Bortle K, Spacek D, Hess GT, Shamim MS, Machol I, Love MI, Aiden EL, Bassik MC, Snyder MP. Static and Dynamic DNA Loops form AP-1-Bound Activation Hubs during Macrophage Development. <strong>Mol Cell. 2017 </strong> 67(6), 1037-1048.
<p>
20. Doynova MD, Markworth JF, Cameron-Smith D, Vickers MH, O Sullivan JM. Linkages between changes in the 3D organization of the genome and transcription during myotube differentiation in vitro. <strong> Skelet Muscle 2017 </strong>. 5;7(1):5
<p>
21. Jiang Y, Loh YE, Rajarajan P, Hirayama T, Liao W, Akbarian S, et al. The methyltransferase SETDB1 regulates a large neuron-specific topological chromatin domain. <strong> Nat Genet. 2017 </strong>49(8):1239-1250.
<p>
22. Bonev B, Mendelson Cohen N, Szabo Q, Fritsch L, Papadopoulos GL, Lubling Y, Xu X, Lv X, Hugnot JP, Tanay A, Cavalli G. Multiscale 3D Genome Rewiring during Mouse Neural Development. <strong>Cell. 2017 </strong> 171(3):557-572.e24.

<p>If you have any questions or comments, please contact us at 
<img src="img/feng_email.Png"> 
</p>
</div>
</div>
<script src="http://www.google-analytics.com/urchin.js" type="text/javascript"> 
</script>
<script type="text/javascript"> 
_uacct = "UA-31923273-1";
urchinTracker();
</script>
</body>
</html>
